toffy/normalize.py

```diff
@@ -641,9 +653,15 @@ def normalize_fov(img_data, norm_vals, norm_dir, fov, channels, extreme_vals):
                       'value for fov {}. Manual inspection for accuracy is '
                       'recommended: {}'.format(fov, bad_channels))
 
-    # correct images and save
+    # correct images and save, ensure that no division by zero happens
     norm_vals = norm_vals.astype(img_data.dtype)
-    normalized_images = img_data / norm_vals.reshape((1, 1, 1, len(norm_vals)))
+    norm_div = norm_vals.reshape((1, 1, 1, len(norm_vals)))
+    normalized_images = np.divide(
+        img_data,
+        norm_div,
+        out=np.zeros_like(img_data, dtype=img_data.dtype),
+        where=norm_div != 0
+    )
```

Comment:
One alternative method that could be used here is [`masked arrays`](https://numpy.org/doc/stable/reference/maskedarray.html). With this we can create a mask over the `norm_vals` array which will filter values of interest. In this case, it would be `0`.

For example:

```python
import numpy as np
norm_vals_masked = np.ma.masked_values(
	x = norm_vals, # array to mask
	value = 0, # the value to mask,
)
```

This would effectively block `0` values from getting passed into`norm_func` on lines 727.